### PR TITLE
[MI-1624] Added DEPLOY.md and CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by kcasas**
/cc @zendesk/mintegrations @zendesk/sustaining @z3n-badam

### Description
Added CONTRIBUTING.md and DEPLOY.md

### References
* JIRA: https://zendesk.atlassian.net/browse/MI-1624

### Risks
* Might have missed some information. Information might be outdated in the future

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Contributing
+
+1. Fill out the Pull Request template, including JIRA issue and risks sections.
+1. cc @zendesk/mintegrations
+1. Get at least 2 :+1:s, including at least 1 from a @zendesk/mintegrations member.
+1. Wait for Travis to be 💚.
```
Comment (samgavinio) [inline CONTRIBUTING.md]: Not true since these poor little apps don't have any tests. 😢 

✅ though.

File: DEPLOY.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+1. Go to [Samson Production](https://samson.zende.sk/projects/magento_app/stages/production) and deploy the healthy tag.
+2. Wait until the deploy is done successfully.
+3. Install app on Zendesk account and do the necessary tests.
+
+## Recovery
+
+In cases of bad deploy:
+1. Revert PR or create another PR that removes the defective change.
+2. Wait for review and two :+1:'s before merge.
```
Comment (gfontanilla) [inline DEPLOY.md]: Nitpicking but the wording makes it seem like the review and 👍s are two separate things.

File: DEPLOY.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+1. Go to [Samson Production](https://samson.zende.sk/projects/magento_app/stages/production) and deploy the healthy tag.
+2. Wait until the deploy is done successfully.
+3. Install app on Zendesk account and do the necessary tests.
+
+## Recovery
+
+In cases of bad deploy:
+1. Revert PR or create another PR that removes the defective change.
+2. Get two :+1:'s before merge.
```
Comment (gfontanilla) [inline DEPLOY.md]: As mentioned in other projects, two +1s for reverting a bad deploy is too restrictive. Can you please change to require just one +1?

**Comment by gfontanilla** [comment]
👍

**Pull request merged**